fc_karpaty/src/main/java/tytarchuk/ProductOverviewPage.java

```diff
@@ -0,0 +1,26 @@
+package tytarchuk;
+
+
+import com.codeborne.selenide.Selenide;
+
+public class ProductOverviewPage {
+    public ProductOverviewPage chooseQuantity(int quantity){
+        Selenide.$x("//select[@id  = 'cart_product_count']").selectOption(quantity-1);
```

Comment:
-1?

Reply:
-1 is for convinience, cause its easier to tape 7 instead of 6 if you want to choose July